schema/bom-1.3-SNAPSHOT.schema.json

```diff
@@ -77,6 +77,14 @@
       "uniqueItems": true,
       "title": "Dependencies",
       "description": "Provides the ability to document dependency relationships."
+    },
+    "compositions": {
+      "$id": "#/properties/compositions",
+      "type": "array",
+      "items": {"$ref": "#/definitions/compositions"},
+      "uniqueItems": true,
+      "title": "Compositions",
+      "description": "Compositions describe constituent parts (including components, services, and dependency relationships) and their completeness."
     }
```

Comment:
I'm wondering, given JSON format limitations, if compositions should be an object with an array of items as a property.

Reply:
What the PR is saying is that compositions is an array of composition objects (I should change this from compositions to composition). The composition is an object, I think we'll put all the composition information in there. I don't know if we would ever put non-composition objects in the compositions array though.

Reply:
Yep, what I'm proposing is something like this in the JSON BOM

```
compositions: {
    items: []
```

Just in case we think we might want to add any additional attributes to compositions element in the future.

Just a thought, I don't feel strongly about it at all.